src/dev/flang/be/c/Intrinsics.java

```diff
@@ -143,7 +143,21 @@ public class Intrinsics extends ANY
             );
         }
         );
-    put("fuzion.std.fileio.exists"       , noFileIo); // NYI: #158
+    put("fuzion.std.fileio.exists"       ,  (c,cl,outer,in) ->
+        {
+          var fileIdent = new CIdent("f");
+          return CStmnt.seq(
+            CExpr.decl("FILE *", fileIdent, CExpr.call("fopen", new List<>(A0.castTo("char *"),CExpr.string("r")))),
+            // Testing if fopen was successful hence file/dir exists
+            CExpr.iff(CExpr.notEq(fileIdent ,new CIdent("NULL")), 
+            CExpr.iff(CExpr.eq(CExpr.call("fclose", new List<>(fileIdent)), CExpr.int8const(0)), CExpr.int8const(1).ret())),
```

Comment:
@fridis I close the stream when the fopen was successful otherwise there is nothing to close.

Reply:
sorry, did not see this.